crates/cli/src/codegen/mod.rs

```diff
@@ -333,7 +358,11 @@ impl Generator {
             CodeGenType::Static => {
                 // Remove no longer necessary exports.
                 module.exports.remove("canonical_abi_realloc")?;
-                module.exports.remove("eval_bytecode")?;
+                // `eval_bytecode` is not present in user plugins and `remove`
+                // returns an error if the export is not present.
+                if module.exports.get_func("eval_bytecode").is_ok() {
+                    module.exports.remove("eval_bytecode")?;
+                }
```

Comment:
This is true because we know the plugin API surface, however, I don't think anything is preventing a plugin from exporting a function named `eval_bytecode` for their own purposes. Probably unlikely, but not impossible. I'd suggest checking if the plugin is not a user provided plugin before removing this export.

Reply:
I've updated the check